my-wireframe/articles/why-do-developers-need-git.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Why do developers need Git?</title>
+</head>
+
+<body>
+    <section>
```

Comment:
Make sure you always have an h1 on your page. Here's a great page from a11y on this https://www.a11yproject.com/posts/how-to-accessible-heading-structure/